Content.Client/Guidebook/Controls/GuideLawsetEmbed.xaml.cs

```diff
@@ -0,0 +1,108 @@
+using System.Diagnostics.CodeAnalysis;
+using Content.Client.Guidebook.Richtext;
+using Content.Client.Message;
+using Content.Client.UserInterface.ControlExtensions;
+using JetBrains.Annotations;
+using Robust.Client.AutoGenerated;
+using Robust.Client.Graphics;
+using Robust.Client.UserInterface;
+using Robust.Client.UserInterface.Controls;
+using Robust.Client.UserInterface.XAML;
+using Robust.Shared.Prototypes;
+
+using Content.Shared.Silicons.Laws;
+using Robust.Shared.Utility;
+using Content.Shared.Contraband;
+
+namespace Content.Client.Guidebook.Controls;
+
+/// <summary>
+/// Control for embedding an AI Lawset in a guidebook
+/// </summary>
+[UsedImplicitly, GenerateTypedNameReferences]
+public sealed partial class GuideLawsetEmbed : Control, IDocumentTag, ISearchableControl, IPrototypeRepresentationControl
+{
+    [Dependency] private readonly IPrototypeManager _prototype = default!;
+
+    private ISawmill _logging = default!;
+
+    public IPrototype? RepresentedPrototype { get; private set; }
+
+    public GuideLawsetEmbed()
+    {
+        RobustXamlLoader.Load(this);
+        IoCManager.InjectDependencies(this);
+        MouseFilter = MouseFilterMode.Stop;
+    }
+
+    public GuideLawsetEmbed(string lawset) : this()
+    {
+        GenerateControl(_prototype.Index<SiliconLawsetPrototype>(lawset));
+    }
+
+    public GuideLawsetEmbed(SiliconLawsetPrototype lawset) : this()
+    {
+        GenerateControl(lawset);
+    }
+
+    private void GenerateControl(SiliconLawsetPrototype lawset)
+    {
+        RepresentedPrototype = lawset;
+        FindControl<PanelContainer>("NameBackground").PanelOverride = new StyleBoxFlat
+        {
+            BackgroundColor = new(22, 22, 140)
+        };
```

Comment:
Keep this in XAML. Don't pre-emptively code for a future use-case that might not exist.